specs/trust-store-trust-policy.md

```diff
@@ -245,7 +245,7 @@ Signature verification levels provide Notary v2 defined behavior for each valida
 #### Selecting a trust policy based on artifact URI
 
 - For a given artifact there MUST be only one applicable trust policy, except for trust policy with global scope.
-- For a given artifact, if there is no applicable trust policy then Notary v2 MUST consider the artifact as untrusted and fail signature verification.
+- For a given artifact, if there is no applicable trust policy then Notary Project specification implementation MUST consider the artifact as untrusted and fail signature verification.
```

Comment:
For a given artifact, if there is no applicable trust policy then the Notary Project specification implementation

Reply:
update to `implementations of the Notary Project specification`